src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/Microsoft.CodeAnalysis.Analyzer.Testing.csproj

```diff
@@ -21,17 +21,18 @@
     </When>
     <When Condition="'$(TargetFramework)' == 'net46'">
       <PropertyGroup>
-        <NuGetApiVersion>4.9.6</NuGetApiVersion>
+        <NuGetApiVersion>4.5.3</NuGetApiVersion>
```

Comment:
Why the downgrades here?

Reply:
They didn't release an update to 4.9.6 that is compatible with this TFM.